assets/src/edit-story/elements/image/edit.js

```diff
@@ -46,20 +47,24 @@ const Element = styled.div`
 
 const FadedImg = styled.img`
   position: absolute;
-  opacity: 0.4;
+  opacity: ${({ opacity }) =>
+    opacity ? opacity * MEDIA_MASK_OPACITY : MEDIA_MASK_OPACITY};
   pointer-events: none;
   ${imageWithScale}
 `;
 
 const CropImg = styled.img`
   position: absolute;
+  opacity: ${({ opacity }) =>
+    opacity ? opacity - opacity * MEDIA_MASK_OPACITY : null};
```

Comment:
What's this doing again? `MEDIA_MASK_OPACITY` is 0.4. So if image opacity is set to 0.5, then the crop image opacity is set to `0.5 - 0.5 * 0.4` = `0.3`. And if it's 0.9, it's set to `0.54`.

Is that the expected outcome?

If it is, it could be rewritten as:

```js
    opacity ? opacity * (1 - MEDIA_MASK_OPACITY) : null};
```

But i'm not completely sure I understand why it's multiplied by the inverse of the mask opacity at all?

Reply:
Oh, is it so that the combined opacity of this image and the overlapping faded image resolves to the proper opacity? And does that work? Are opacities additive? .2 image on top of similar .2 image equals a .4 image?

Reply:
Yep, that's the goal and it should work. At least testing didn't show otherwise! Perhaps a comment would be appropriate.

Reply:
Nope, that doesn't work. [Opacities are inversely multiplicative](https://codepen.io/barklund/pen/XWbaZav).

Thus it should be (I think):

```js
    opacity ? 1 - ((1 - opacity) / (1 - (opacity * MEDIA_MASK_OPACITY))) : null};
```